res/res_pjsip/config_transport.c

```diff
@@ -1246,6 +1311,34 @@ static int transport_tls_method_handler(const struct aco_option *opt, struct ast
 	return 0;
 }
 
+static int transport_tcp_keepalive_bool_handler(const struct aco_option *opt, struct ast_variable *var, void *obj) {
```

Comment:
This does not need to be implemented. The configuration framework provides an OPT_BOOL_T which implements this.